ingress/controllers/nginx/main.go

```diff
@@ -41,6 +41,10 @@ const (
 )
 
 var (
+	// value overwritten during build. This can be used to resolve issues.
+	version = "0.0"
```

Comment:
It's helpful when people give you logs and don't know if they changed the image on the rc/restart pods etc, or even if they're using latest